diff --git forkSrcPrefix/src/libbluray/disc/dec.c forkDstPrefix/src/libbluray/disc/dec.c
index 73f64a3760c9f49431599e6775620b6e21b53573..73bdcee79d8a3f6c6cc0ab2d4711cd50fc138e4e 100644
--- forkSrcPrefix/src/libbluray/disc/dec.c
+++ forkDstPrefix/src/libbluray/disc/dec.c
@@ -284,7 +284,7 @@ static void _dec_load(BD_DEC *dec, BD_ENC_INFO *i)
  *
  */
 
-BD_DEC *dec_init(struct dec_dev *dev, BD_ENC_INFO *enc_info,
+BD_DEC *bd_dec_init(struct dec_dev *dev, BD_ENC_INFO *enc_info,
                  const char *keyfile_path,
                  void *regs, void *psr_read, void *psr_write)
 {
diff --git forkSrcPrefix/src/libbluray/disc/disc.c forkDstPrefix/src/libbluray/disc/disc.c
index d5e39c491cc70db5d7eb7abc22d15cbf9c89e242..efe3fcd9a96332eae40458e142d4fb95b2fe7bb5 100644
--- forkSrcPrefix/src/libbluray/disc/disc.c
+++ forkDstPrefix/src/libbluray/disc/disc.c
@@ -348,7 +348,7 @@ BD_DISC *disc_open(const char *device_path,
     }
 
     struct dec_dev dev = { p->fs_handle, p->pf_file_open_bdrom, p, (file_openFp)disc_open_path, p->disc_root, device_path };
-    p->dec = dec_init(&dev, enc_info, keyfile_path, regs, psr_read, psr_write);
+    p->dec = bd_dec_init(&dev, enc_info, keyfile_path, regs, psr_read, psr_write);
 
     return p;
 }
diff --git forkSrcPrefix/src/libbluray/disc/dec.h forkDstPrefix/src/libbluray/disc/dec.h
index d1329be272beb1cba4d9809be9ca343303ca4b7c..72d3ae1aae25ef05dcd8a189e89c546b091763b3 100644
--- forkSrcPrefix/src/libbluray/disc/dec.h
+++ forkDstPrefix/src/libbluray/disc/dec.h
@@ -45,7 +45,7 @@ struct dec_dev {
 
 typedef struct bd_dec BD_DEC;
 
-BD_PRIVATE BD_DEC *dec_init(struct dec_dev *dev,
+BD_PRIVATE BD_DEC *bd_dec_init(struct dec_dev *dev,
                             struct bd_enc_info *enc_info,
                             const char *keyfile_path,
                             void *regs, void *psr_read, void *psr_write);
